### DISCUSSION TITLE
Deploying Client on AWS

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#3144] [Hubs Cloud Questions] [answered]
Still fighting with this... Any help would be appreciated.

```
$ npm run start

> hubs@0.0.1 start C:\Git\hubs
> webpack-dev-server --mode=development --env.loadAppConfig
Project is running at https://localhost:8080/
webpack output is served from /
404s will fallback to /index.html
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
```

Basically I would like to know what must be done to see https://localhost:8080/ on a Windows 10 desktop

Thanks

**Comment by johnshaughnessy** [comment, marked as answer]
@ocularvr Can you provide more details about what is going wrong? Are you running `npm run start` after `npm run login` in order to [test a custom client against your hubs cloud server](https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html)? If all goes well, you should be able to connect to https://localhost:8080/ after the `npm run start` command finishes compilation and the webpack dev server starts. 
If you are able to start the server but can't connect to it -- what version of node/npm are you running? (`node --version`) We just found [this issue](https://github.com/mozilla/hubs/discussions/3147#discussioncomment-94316) with older versions of node

**Reply by ocularvr** [reply]
Hi, its all good now. I managed to run the local client after updating node to v12.19.0. 
One more question while were on the subject, if I have a running hubs on aws and I deploy my local client to my aws server, will I lose all of the information on my aws server (rooms, identities, etc). 
Thanks and please tell Robin that I managed to do it finally :)
